bitnami/discourse/README.md

```diff
@@ -203,7 +205,9 @@ The following table lists the configurable parameters of the Discourse chart and
 |-------------------------------------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------|
 | `redis.enabled`                           | Deploy Redis container(s)                                                             | `true`                                                       |
 | `redis.usePassword`                       | Use password authentication                                                           | `false`                                                      |
-| `redis.password`                          | Password for Redis authentication                                                     | `nil`                                                        |
+| `redis.password`                          | Password for Redis authentication  - ignored if existingSecret is set to true                                                    | `my-password`                                                        |
```

Comment:
The re-wording I mentioned also applies here 😁

Also, `my-password` should be updated to `nil`.